service/src/main/java/com/epam/esm/data/GiftCertificateDto.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.data;
+
+import com.epam.esm.entities.Tag;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Data
+public class GiftCertificateDto {
+
+    private int id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Double duration;
+    private String createDate;
+    private String lastUpdateDate;
```

Comment:
To make it easier for you to work with these variables, you can do the following:
![image](https://user-images.githubusercontent.com/49719217/132675856-9e2fe1e8-426c-43f4-8d00-ca692c03acae.png)

Reply:
I thougt it can be done with annotations, but didn't found this one. Thank you